.github/workflows/oneshot_candidate_release.yml

```diff
@@ -0,0 +1,56 @@
+name: Schedule candidate release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  tag_release:
+    name: "Tag candidate release"
+    runs-on: ubuntu-18.04
+    # Don't run this in everyone's forks.
+    if: github.repository == 'google/iree' || github.repository == 'NodLabs/SHARK'
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+
+      - name: Compute version
+        run: |
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+          package_version="$(printf '%(%Y%m%d)T.${{ github.run_number }}')"
+          tag_name="candidate-${package_version}"
+          echo "package_version=${package_version}" >> $GITHUB_ENV
+          echo "tag_name=${tag_name}" >> $GITHUB_ENV
+
+      - name: Updating candidate tag
+        run: |
+          git tag "${tag_name}"
+
+      - name: Pushing changes
+        uses: ad-m/github-push-action@v0.6.0
+        with:
+          github_token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          branch: ${{ env.ref }}
+          tags: true
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.WRITE_ACCESS_TOKEN }}
+        with:
+          tag_name: ${{ env.tag_name }}
+          release_name: iree candidate ${{ env.tag_name }}
+          body: |
+            Automatic candidate release of iree.
+          draft: true
+          prerelease: true
+
+      - name: "Invoke workflow :: Build Native Release Packages"
```

Comment:
We do have some plans to migrate these workflows from GitHub Actions to Buildkite (first we'll migrate our presubmit/postsubmit pipelines, then release workflows, so it'll be at least a few months before much changes here specifically I think).

I'd be curious to hear more about your use case to understand why using the workflows from this repository in your downstream fork is needed.

For example, _testing_ the release pipeline from a fork for contributions back to the upstream repo is a case we want to support, but regularly generating the full set of python wheels with forked code and releasing them would raise some questions :)

Reply:
(Downstream releases are fine, but there are things like renaming "iree" to "shark" or "iree with experimental features" that I don't see the upstream workflows as being set up for - at that point creating a new release pipeline that builds what you want how you want seems better)

Reply:
ideally we don't want any downstream fork of IREE. The only thing we are carry indefinitely are the CUDA caching allocator but hopefully that will also morph into something else upstream. I plan to use a different version tag (the -suffix) seems to work. 

Overall we want to provide a way to replace stock IREE with this version via pip if possible. If we have to create a new package name and namespace then we have to create a "deep fork" to rename the namespace etc and we would rather not. 

We want downstream users (and other customers to be able to install by pointing to something like https://github.com/NodLabs/SHARK/releases/tag/untagged-bb5737a2d500d03df898 )

Reply:
also it may be helpful for customers building downstream vendorized versions if we keep the release in github actions.